src/cmd/display.rs

```diff
@@ -78,8 +96,6 @@ pub async fn invoke(args: &[String], ctx: &mut Context) -> Result<String> {
     stdout().execute(LeaveAlternateScreen)?;
     disable_raw_mode()?;
 
-    ctx.lh.stop(&path.as_lh_vec()).await?;
```

Comment:
Note that this `stop` is no longer needed, since stopping now happens implicitly when the stream is dropped (with the same request id as the initial request).